app/models/user.rb

```diff
@@ -265,6 +265,11 @@ def questions
     Node.questions.where(status: 1, uid: id)
   end
 
+  def content_followed_in_past_period(time_period)
+     nodes_created = self.node.select {|node| Time.at(node.created) >= time_period }
+     nodes_edited = self.node.select {|node| Time.at(node.changed) >= time_period unless node.changed.empty?}
```

Comment:
Here, looks like you may want to use an ActiveRecord query like:

https://github.com/publiclab/plots2/blob/master/app/controllers/stats_controller.rb#L15

```ruby
@notes = Node.select(%i[created type status])
                 .where(type: 'note', status: 1, created: @start.to_i..@end.to_i)
```